### PR TITLE
RHPAM-3940 kafka version to 2.8.1

### PR DESCRIPTION
**Pull request by Ginxo**
**Thank you for submitting this pull request**

**JIRA**: https://issues.redhat.com/browse/RHPAM-3940

I had to upgrade `scala-collection-compat_2.12:2.3.0` coming from new `kafka:2.8.1` to  `scala-collection-compat_2.12:2.6.0` due to 
```
[WARNING] Rule 0: org.apache.maven.plugins.enforcer.BanDuplicateClasses failed with message:
Duplicate classes found:

  Found in:
    org.scala-lang.modules:scala-collection-compat_2.12:jar:2.3.0:test
    org.scala-lang:scala-library:jar:2.12.13:test
  Duplicate classes:
    scala/annotation/nowarn.class
    scala/annotation/nowarn$.class
```

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1895
* https://github.com/kiegroup/droolsjbpm-integration/pull/2723

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: drools-ha/pom.xml [hunk 1]
```diff
@@ -24,7 +24,6 @@
     <surefire.forkCount>1</surefire.forkCount>
     <version.javax.el>3.0.1-b08</version.javax.el>
     <version.javax.ws.rs>2.0.0.Final</version.javax.ws.rs>
-    <version.org.apache.kafka>2.2.1</version.org.apache.kafka>
```
Comment (Ginxo) [inline drools-ha/pom.xml:27]: `version.org.apache.kafka` taken from kie-parent project

**Comment by mareknovotny** [comment]
enforcer reports duplicated classes in two deps
```
[2022-03-01T09:09:22.282Z] [WARNING] Rule 0: org.apache.maven.plugins.enforcer.BanDuplicateClasses failed with message:
[2022-03-01T09:09:22.282Z] Duplicate classes found:
[2022-03-01T09:09:22.282Z] 
[2022-03-01T09:09:22.282Z]   Found in:
[2022-03-01T09:09:22.282Z]     org.scala-lang.modules:scala-collection-compat_2.12:jar:2.3.0:test
[2022-03-01T09:09:22.282Z]     org.scala-lang:scala-library:jar:2.12.13:test
[2022-03-01T09:09:22.282Z]   Duplicate classes:
[2022-03-01T09:09:22.282Z]     scala/annotation/nowarn.class
[2022-03-01T09:09:22.282Z]     scala/annotation/nowarn$.class
```

**Comment by Ginxo** [comment]
Test errors are random and not related with the change itself. Can you please @kiegroup/gatekeepers merge this set of PRs?

**Comment by mareknovotny** [comment]
@triceo you missed the parent PR, which anyway shows FDB in failures and not finished :(

**Pull request merged**